src/maintainer/adding_pkgs.rst

```diff
@@ -28,8 +28,19 @@ There are multiple ways to get started:
    start by using the `conda-forge helper script for R recipes <https://github.com/bgruening/conda_r_skeleton_helper>`_ instead.
    Then if necessary, you can make manual edits to the recipe.
 #. If it is a python package, you can generate the recipe as a starting point with ``grayskull``.
-   Use ``conda install -c conda-forge grayskull`` to install ``grayskull``, followed by ``grayskull pypi your_package_name`` to generate the recipe. Note that you do *not* necessarily have to use ``grayskull``, and the
-   recipes produced by ``grayskull`` might need to be reviewed and edited. Read more about ``grayskull`` and how to use it `here <https://github.com/conda-incubator/grayskull#introduction>`__.
+
+.. note::
+
+  `Grayskull <https://github.com/conda-incubator/grayskull>`_ is an automatic conda recipe generator. The goal of this project is to generate concise recipes
+  for conda-forge and eventually replace conda skeleton. Presently, Grayskull can generate recipes for Python packages available on PyPI and also those not published on PyPI and are only available as GitHub repositories.
+
+  Installation and usage of ``grayskull``:
+
+  - Run ``conda install -c conda-forge grayskull`` to install ``grayskull``.
+  - Followed by ``grayskull pypi your_package_name`` to generate the recipe.
```

Comment:
```suggestion
  - Followed by ``grayskull pypi --strict-conda-forge your_package_name`` to generate the recipe.
```

This adds a couple of optional settings that are helpful in a conda-forge context.

Reply:
Good to know that @BastianZim! I have updated the PR with these changes.